test/e2e/single_api/tts_pipeline/test_multi_synthesis.py

```diff
@@ -63,14 +69,11 @@ def test_post_multi_synthesis_200(client: TestClient) -> None:
     ]
     response = client.post("/multi_synthesis", params={"speaker": 0}, json=queries)
     assert response.status_code == 200
-
-    # FileResponse 内の zip ファイルに圧縮された .wav から抽出された音声波形が一致する
-    # FIXME: スナップショットテストを足す
-    # NOTE: ZIP ファイル内の .wav に Linux-Windows 数値精度問題があるため解凍が必要
     assert response.headers["content-type"] == "application/zip"
-    # from test.utility import summarize_wav_bytes
-    # from syrupy.assertion import SnapshotAssertion
-    # # zip 解凍
-    # wav_summarys = map(lambda wav_byte: summarize_wav_bytes(wav_byte), wav_bytes)
-    # wavs_summary = concatenate_func(wav_summarys)
-    # assert snapshot == wavs_summary
+
+    # zipに含まれる全てのwavの波形がスナップショットと一致することを確認
```

Comment:
現状の雰囲気的に前後にスペースを開ける規則になってそうなことに気づいたので、ちょっと変更させていただきます！
あとコメントもちょっとだけsynthesisのに寄せてみました。ぶっちゃけ元のコメントで全く問題ないですが･･･！
```suggestion
    # zip 内の全ての wav の波形がスナップショットと一致する
```